src/widget/form/profileform.cpp

```diff
@@ -46,42 +70,59 @@ IdentityForm::IdentityForm() :
     
     bodyUI->toxGroup->layout()->addWidget(toxId);
 
+    profilePicture = new MaskablePixmapWidget(this, QSize(64, 64), ":/img/avatar_mask.png");
+    profilePicture->setPixmap(QPixmap(":/img/contact_dark.png"));
+    profilePicture->setClickable(true);
+    connect(profilePicture, SIGNAL(clicked()), this, SLOT(onAvatarClicked()));
+    QHBoxLayout *publicGrouplayout = qobject_cast<QHBoxLayout*>(bodyUI->publicGroup->layout());
+    publicGrouplayout->insertWidget(0, profilePicture);
```

Comment:
I take this back. Although feasible, by modifying the MaskablePixmapWidget to not require constructor arguments, the main profile picture is also handled this way. Oh well...